doc/requirements.txt

```diff
@@ -0,0 +1,4 @@
+numpy >= 1.17.0
```

Comment:
What is this file for?  In principle, these dependencies should be installed when running `python -m pip install -e ".[tests, doc]"` because of
https://github.com/giotto-ai/pyflagser/blob/39f40a48109c3297da7115c36369903221adc238/setup.py#L57-L61

Reply:
Yes if you install the package `pyflagser`, same point as stated previously. This file allows to make the documentation almost independent from the package.
But maybe, it over engineering and we could assume that documentation requires that `pyflagser` is installed ?

Reply:
I think that the philosophy always was that the package should be installed if the documentation is to be built.  I like the idea of having all the logic only in `setup.py`, where there are `EXTRAS_REQUIRE` with `'doc'` and `'tests'`.  It would be easier to maintain in my opinion.  On the other hand, maybe I am missing something related to the CI?

Reply:
Oh no wait, I get it.  This way, people can build the docs even if they don't have the developer install (similarly to how we did a lot of work to make the tests work with the wheels).  Is that it?

Reply:
Exactly people then can build the docs without the developer install, documentation and the package are decoupled.
But I don't have a solid opinion on this, I'm okay with in order to build the documentation you need the dev install.

Reply:
I'm fine with this approach.